### PR TITLE
[INF-93] disable rsyslog in favor of logspout

### PR DESCRIPTION
**Pull request by joaquincasares**
Sister PR to https://github.com/AudiusProject/audius-protocol/pull/3124

### CODE REVIEW

File: creator-node/prod.env [hunk 1]
```diff
@@ -28,3 +28,4 @@ delegateOwnerWallet=
 delegatePrivateKey=
 discoveryProviderWhitelist=
 spOwnerWallet=
+enableRsyslog=0
```
Comment (cheran-senthil) [inline creator-node/prod.env:31]: can we use true / false instead?

Comment (joaquincasares) [reply creator-node/prod.env:31]: Hey! I thought about that and we're having that conversation here:

https://github.com/AudiusProject/audius-protocol/pull/3124#discussion_r882909502

Comment (cheran-senthil) [reply creator-node/prod.env:31]: Also, required values are at the bottom and left unset, pls move this to the above section, thanks

File: creator-node/prod.env [hunk 2]
```diff
@@ -6,6 +6,7 @@ dbConnectionPoolMax=500
 redisHost=cache
 redisPort=6379
 
+audius_enable_rsyslog=false
```
Comment (cheran-senthil) [inline creator-node/prod.env:9]: I think this one is supposed to be camelCase while discovery is snake_case

Thanks for the comment separation

Comment (joaquincasares) [reply creator-node/prod.env:9]: I did notice that. Why do we do camelCasing in creatorNode and not in discovery_provider?

And sure thing!

Comment (cheran-senthil) [reply creator-node/prod.env]: Because creator node is written in java script and discovery provider is written in python

**Review by cheran-senthil — APPROVED** [review]

**Pull request merged**